.github/workflows/clear-cache.yml

```diff
@@ -12,7 +12,11 @@ jobs:
     name: Clear all cache
     runs-on: ubuntu-latest
     steps:
+      - name: Checkout code
```

Comment:
do we need to checkout the repo in order to clear its cache?

Reply:
Yeah, or to give remote name